.devcontainer/Dockerfile

```diff
@@ -344,7 +344,7 @@ RUN ${PYTHON_VENV_EXECUTABLE} -m pip install --quiet --upgrade --no-cache-dir "s
 ### install python3-aioeventlet and bcc-tools from the magma apt repo
 COPY orc8r/tools/ansible/roles/pkgrepo/files/jfrog.pub /tmp/
 RUN apt-key add /tmp/jfrog.pub && \
-    echo "deb https://facebookconnectivity.jfrog.io/artifactory/list/dev-focal/ focal main" > /etc/apt/sources.list.d/fbc.list && \
+    echo "deb https://artifactory.magmacore.org/artifactory/debian-test focal-ci main" > /etc/apt/sources.list.d/fbc.list && \
```

Comment:
NIT: the magmacore repo is still written into fbc.list - this applies for multiple changes in this PR